src/main/resources/filter/web/positioning.filter

```diff
@@ -7,6 +7,11 @@ attribute=bottom
 attribute=right
 attribute=left
 
+attribute=min-height
+attribute=min-width
```

Comment:
We can directly add all relevant attributes of https://developer.mozilla.org/en-US/docs/Web/CSS/min-width (see navigation), mainly:
```
width
height
max-width
max-height
```